recipes-python/pyarrow/apache-arrow_git.bb

```diff
@@ -0,0 +1,71 @@
+require arrow.inc
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/build"
+
+PARQUET_TEST_DATE="${S}/cpp/submodules/parquet-testing/data"
+ARROW_TEST_DATA="${S}/testing/data"
+ARROW_HOME="/usr"
+
+inherit python3native cmake clang
+
+DEPENDS += " \
+    boost \
+    boost-native \
+    python3-cython \
+    python3-cython-native \
+    python3-numpy \
+    python3-numpy-native \
+    python3-setuptools-scm \
+    python3-setuptools-scm-native \
+    python3-setuptools \
+    python3-setuptools-native \
+    "
+
+SRC_URI += " file://CMakeSettings.json "
+
+EXTRA_OECMAKE += "\
+    -DCMAKE_INSTALL_PREFIX=${ARROW_HOME} \
+    -DARROW_JEMALLOC=OFF \
+    -DARROW_PYTHON=ON \
+    -DARROW_BUILD_STATIC=ON \
+    -DARROW_COMPUTE=ON \
+    -DARROW_CSV=ON \
+    -DARROW_JSON=ON \
+    -DARROW_FILESYSTEM=ON \
+    -DARROW_PARQUET=ON \
+    -DPARQUET_BUILD_EXECUTABLES=ON \
+    -DARROW_WITH_ZLIB=ON \
+    -DARROW_PYTHON=ON \
+    -DARROW_DATASET=ON \
+"
+
+OECMAKE_SOURCEPATH = "${S}/cpp"
+
+do_configure:prepend () {
+    export ARROW_HOME="${ARROW_HOME}"
+    export CMAKE_TOOLCHAIN_FILE="${WORKDIR}/toolchain.cmake"
+    mkdir -p "${ARROW_HOME}"
+    sed "s,WORKDIRTAG,${WORKDIR},g" "${WORKDIR}/CMakeSettings.json" > "${S}/cpp/CMakeSettings.json"
+    sed -i "s,YOCTO_BUILDDIR,${B},g" "${S}/cpp/CMakeSettings.json"
+    sed -i "s,YOCTO_INSTALLDIR,${D},g" "${S}/cpp/CMakeSettings.json"
```

Comment:
A single `sed` command with multiple `-e` options would be better.

Also, instead of being `aarch64` specific, something like `${TARGET_ARCH}` should be used in the same `sed` command  for the `name` and `inheritEnvironments` JSON tags.